src/app/components/AutoComplete.svelte

```diff
@@ -0,0 +1,188 @@
+<script lang="ts">
+  // TODO: the following interface should be removed once #33 is done
+  interface dataObj {
+    bcp47Tag: string;
+    language: string;
+  }
+  enum keyboardKey {
+    Up = "ArrowUp",
+    Down = "ArrowDown",
+    Enter = "Enter",
+  }
+  export let label = "";
+  export let subtext = "";
+  // Testing Data
+  export let results: dataObj[] = [];
+  // To store Filted array
+  let filetred: dataObj[] = [];
```

Comment:
Typo:

```suggestion
  // To store filtered array
  let filtered: dataObj[] = [];
```

It will also need to be changed throught the rest of the file!

Reply:
omg! I wish I could Grammarly my code 😭😂